### DISCUSSION TITLE
Need a better way to install on a server

### DISCUSSION THREAD

**Discussion by stantechie** [OpnForm/OpnForm#575] [General]
I have a Plesk server and I've tried the Local Deployment, but it fails at the git clone - need to use https://github instead of git@github since that doesn't work. Got past that but because I'm not using herd and this is on an actual server, I don't know how to configure MySQL/MariaDB or anything beyond that, the documentation is limited.
Also tried the Docker route, which I got spun up and running using the jhumanj/opnform (which says not to use). It loads, but I am unable to login with admin@opnform.com/password - is the docker version using a different default password?
I'd really like to use this to build some forms, however it feels like it's built to fail so that people go to the cloud version. I'm not sure if that's intentional or not?

**Comment by JhumanJ** [comment]
Hi - have taken a look at the docs? https://docs.opnform.com/deployment/docker

Please don't use the deprecated docker image, but rather the docker compose setup. We spend a lot of time creating this product and the documentation around it, it's always a work in progress. We do not make this "fail" so that people go to the cloud version - in fact I invite you to look at the commit history to see the amount of time recently invested in improving the self-hosted setup & guides. That being said, using the cloud version is the best way to support us.

Please **make sure to read the readme and the docs** - and if anything is unclear remember that this is an open-source project, and that your contribution is more than welcome!
